hypershift-operator/controllers/hostedcluster/internal/platform/aws/aws.go

```diff
@@ -292,11 +292,11 @@ func (AWS) ReconcileSecretEncryption(ctx context.Context, c client.Client, creat
 
 func ValidCredentials(hc *hyperv1.HostedCluster) bool {
 	oidcConfigValid := meta.FindStatusCondition(hc.Status.Conditions, string(hyperv1.ValidOIDCConfiguration))
-	if oidcConfigValid != nil && oidcConfigValid.Status == metav1.ConditionFalse {
+	if oidcConfigValid != nil && oidcConfigValid.Status != metav1.ConditionTrue {
```

Comment:
Can you describe an scenario where this is helping?

Reply:
hmm, I checked the code again and it doesn't seem that `ValidOIDCConfiguration` can have an `Unknown` status, but I think it wouldn't hurt to leave it like this for future proofing.

Reply:
Ohh my bad, I mistook for `ValidAWSIdentityProvider` one.

Reply:
>but I think it wouldn't hurt to leave it like this for future proofing.

It we change it I'd like to make sure there's a valid use case that's justifying the change.

Reply:
fixed